foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/jpa/jpql/EclipseLinkSemanticValidatorHelper.java

```diff
@@ -378,6 +379,19 @@ public boolean isCollectionMapping(Object mapping) {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean isEmbeddableMapping(Object mapping) {
+
+        if (mapping == null) {
+            return false;
```

Comment:
this nulll check can be removed since `null instanceof org.eclipse.persistence.mappings.EmbeddableMapping` is  `false`

Reply:
Makes sense. Removed.